Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -3311,3 +3311,43 @@
 
 /* Info alert when editing a generated column */
 "The current field \"%@\" is a generated column and therefore cannot be edited." = "The current field \"%@\" is a generated column and therefore cannot be edited.";
+
+/* Pin a view to the top of the list of tables
+ Table List : Context Menu : Pin View
+ Table List : Gear Menu : Pin View */
+"Pin View" = "Pin View";
+
+/* Pin a table to the top of the list of tables
+ Table List : Context Menu : Pin Table
+ Table List : Gear Menu : Pin Table */
+"Pin Table" = "Pin Table";
+
+/* Pin a procudure to the top of the list of tables
+ Table List : Context Menu : Pin Procedure
+ Table List : Gear Menu : Pin Procedure */
+"Pin Procedure" = "Pin Procedure";
+
+/* Pin a function to the top of the list of tables
+ Table List : Context Menu : Pin Function
+ Table List : Gear Menu : Pin Function */
+"Pin Function" = "Pin Function";
+
+/* Unpin a view that is currently pinned at the top of the list of tables
+ Table List : Context Menu : Unpin View
+ Table List : Gear Menu : Unpin View */
+"Unpin View" = "Unpin View";
+
+/* Unpin a table that is currently pinned at the top of the list of tables
+ Table List : Context Menu : Unpin Table
+ Table List : Gear Menu : Unpin Table */
+"Unpin Table" = "Unpin Table";
+
+/* Unpin a procudure that is currently pinned at the top of the list of tables
+ Table List : Context Menu : Unpin Procedure
+ Table List : Gear Menu : Unpin Procedure */
+"Unpin Procedure" = "Unpin Procedure";
+
+/* Unpin a function that is currently pinned at the top of the list of tables
+ Table List : Context Menu : Unpin Function
+ Table List : Gear Menu : Unpin Function */
+"Unpin Function" = "Unpin Function";
```

Comment:
No need to add the language strings to all the other languages. I would go ahead and remove them from all the other languages and just set them up for English - we use crowdin to do localization and it will just re-create all the other language files filling in missing ones based on the English one. Keeps things clearer if you only edit the one (English) lang file